core/src/main/resources/templates/activityLog.html

```diff
@@ -224,23 +224,25 @@ <h4 class="mb-3">Help</h4>
 											<table>
 												<tr>
 													<td><i class="mdi mdi-file-document"></i></td>
-													<td><a class="dropdown-item" target="_blank" href="https://klaw-project.io/docs">Documentation</a></td>
+													<td><a class="dropdown-item" target="_blank" rel="noopener noreferrer"
```

Comment:
Just to confirm 

target="_blank" opens a new tab?

 rel="noopener noreferrer" this is an added security measure right so you can't transfer cookies and and stuff to the new site you just opened in the tab?
 
 Changes look good, I remember needing to make them just trying to remind myself of what the changes do exactly again :/

Reply:
Exactly right!